x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -263,6 +263,8 @@ export interface PendingActionsResponse {
 
 export type PendingActionsRequestQuery = TypeOf<typeof ActionStatusRequestSchema.query>;
 
+export const RESPONSE_ACTION_STATUS = ['completed', 'failed', 'pending'] as const;
```

Comment:
yeah. I'll use `successful` instead of `success`. Easier to turn the values to badges that we see on the table and the filter with the logic that is in place.

Reply:
ef49415f8532a5d60fff9f9fd1f3de00070f43d2